searx/plugins/__init__.py

```diff
@@ -86,3 +171,6 @@ def call(self, ordered_plugin_list, plugin_type, request, *args, **kwargs):
 plugins.register(search_on_category_select)
 plugins.register(tracker_url_remover)
 plugins.register(vim_hotkeys)
+# load external plugins
+if 'plugins' in settings:
+    plugins.register(*settings['plugins'], external=True)
```

Comment:
Be aware: this creates folders and copies files when importing the `plugins` package (__init__).